routes/api/search.js

```diff
@@ -67,6 +67,97 @@ module.exports = {
+                    ]
+                }
+         *
+         * @apiSuccess {String} message Success message
+         * @apiSuccess {Object} data Empty object
+         * @apiSuccessExample {object} Success-Response:
+         *      {
+                    "message": "No results found.",
+                    "data": {}
+                }
+         *
+         * @apiError {String} message Error message
+         * @apiError {Object} data empty
+         * @apiErrorExample {object} Error-Response:
+         *      {"message": "Validation failed", "data": {}}
+         */
+        searchRouter.route("/updateStatus").get(
+            Middleware.Auth.ensureAuthenticated(),
+            Middleware.Auth.ensureAuthorized(),
+            Middleware.Validator.Search.statusValidator,
+            Middleware.parseBody.middleware,
+            Middleware.Search.parseQuery,
+            Middleware.Search.executeStatusAction,
+            Controllers.Search.searchResults
+        );
+
+        /**
+         * @api {get} /search/sendEmails execute an action on a specific query for any defined model
+         * @apiName search
+         * @apiGroup Search
+         * @apiVersion 0.0.8
+         *
+         * @apiParam (query) {String} model the model to be searched
+         * @apiParam (query) {Array} q the query to be executed. For more information on how to format this, please see https://docs.mchacks.ca/architecture/
+         * @apiParam (query) {String} model the model to be searched
+         * @apiParam (query) {String} sort either "asc" or "desc"
+         * @apiParam (query) {number} page the page number that you would like
+         * @apiParam (query) {number} limit the maximum number of results that you would like returned
+         * @apiParam (query) {any} sort_by any parameter you want to sort the results by
+         * @apiParam (query) {String} |
+         * @apiParam (query) {boolean} expand whether you want to expand sub documents within the results
+         * 
+         * @apiSuccess {String} message Success message
+         * @apiSuccess {Object} data Results
+         * @apiSuccessExample {object} Success-Response:
+         *      {
+                    "message": "Hacker update emails sent.",
+                    "data": {}
+                }
+         *
+         * @apiSuccess {String} message Error message
+         * @apiSuccess {Object} data Specific Error message
+         * @apiSuccessExample {object} Success-Response:
+         *      {
+                    "message": "Error while generating email",
+                    "data": "Email service failed."
+                }
+         */
+        searchRouter.route("/sendEmails").get(
+            Middleware.Auth.ensureAuthenticated(),
+            Middleware.Auth.ensureAuthorized(),
+            Middleware.Validator.Search.emailValidator,
+            Middleware.parseBody.middleware,
+            Middleware.Search.parseQuery,
+            Middleware.Search.executeEmailAction,
```

Comment:
This endpoint is interesting. The name sounds like a generic 'send emails' route. The middleware names look like generic 'send emails' validators. But 'emailValidator' validates for 'search + status', and then the executeEmailAction middleware ends up calling a service that seems to only send status update emails. I believe there's a mismatch between what this endpoint says it's doing, and what it actually does. 

Are there plans to expand the functionality of the route? I think aligning functionality and naming would help. Also, if this is solely a status email sender, consider (from a comment above):
[...,
Middleware.Validator.Search.searchQueryValidator,
Middleware.Validator.Search.statusValidator,
...]

Does that align more or less with what this endpoint currently does?